card/mod/01_core/spec/set/all/fetch_spec.rb

```diff
@@ -10,196 +10,233 @@
+      expect(Card.cache.soft.store.keys).to eq([])
 
       Card::Auth.as_bot do
-        a = Card.fetch("A")
+        a = Card.fetch('A')
         expect(a).to be_instance_of(Card)
 
         # expires the saved card
         expect(Card.cache).to receive(:delete).with('a').and_call_original
         expect(Card.cache).to receive(:delete).with(/~\d+/).at_least(1)
         # expires plus cards
-        #expect(Card.cache).to receive(:delete).with('c+a')
-        #expect(Card.cache).to receive(:delete).with('d+a')
-        #expect(Card.cache).to receive(:delete).with('f+a')
-        #expect(Card.cache).to receive(:delete).with('a+b')
-        #expect(Card.cache).to receive(:delete).with('a+c')
-        #expect(Card.cache).to receive(:delete).with('a+d')
-        #expect(Card.cache).to receive(:delete).with('a+e')
-        #expect(Card.cache).to receive(:delete).with('a+b+c')
+        # expect(Card.cache).to receive(:delete).with('c+a')
+        # expect(Card.cache).to receive(:delete).with('d+a')
+        # expect(Card.cache).to receive(:delete).with('f+a')
+        # expect(Card.cache).to receive(:delete).with('a+b')
+        # expect(Card.cache).to receive(:delete).with('a+c')
+        # expect(Card.cache).to receive(:delete).with('a+d')
+        # expect(Card.cache).to receive(:delete).with('a+e')
+        # expect(Card.cache).to receive(:delete).with('a+b+c')
 
         # expired including? cards
-        #expect(Card.cache).to receive(:delete).with('x').exactly(2).times
-        #expect(Card.cache).to receive(:delete).with('y').exactly(2).times
+        # expect(Card.cache).to receive(:delete).with('x').exactly(2).times
+        # expect(Card.cache).to receive(:delete).with('y').exactly(2).times
 
         a.save!
       end
     end
+    describe 'default option' do
+      context "when card doesn't exist" do
+        it 'initializes new cards' do
+          card = Card.fetch 'non-existent',
+                            new: { default_content: 'default content' }
+          expect(card.content).to eq 'default content'
+        end
+      end
+      context 'when new card exist' do
+        it "doesn't change anything" do
+          Card.new name: 'new card',
+                   '+sub' => { content: 'some content' }
+          card = Card.fetch 'new card+sub',
+                            new: { default_content: 'new content' }
+          expect(card.content).to eq 'some content'
+        end
+      end
+    end
 
-    describe "preferences" do
+    describe 'preferences' do
       before do
         Card::Auth.current_id = Card::WagnBotID
       end
 
-      it "prefers db cards to pattern virtual cards" do
-        c1=Card.create!(name: "y+*right+*structure", content: "Formatted Content")
-        c2=Card.create!(name: "a+y", content: "DB Content")
-        card = Card.fetch("a+y")
+      it 'prefers db cards to pattern virtual cards' do
+        c1=Card.create!(name: 'y+*right+*structure', content: 'Formatted Content')
```

Comment:
Useless assignment to variable - `c1`.<br>Line is too long. [82/80]<br>Surrounding space missing for operator `=`.